src/views/Pipeline.vue

```diff
@@ -176,6 +179,7 @@
           <JobConfiguration :title="title" :job="currentJob" :status="currentJobStatus" :type="freqType" :key="currentJob"/>
         </aside>
       </main>
+      <ion-loading v-if="isLoading" show-backdrop="true" spinner="crescent" />
```

Comment:
I think this is not need we should add loader the same way we do for other instances

Reply:
Done Sir